autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -42,6 +42,11 @@ const CourseSelectColumn: React.FC = () => {
   React.useEffect(() => {
     if (!term) return;
 
+    // if any course cards are loading, don't try to save
+    for (let i = 0; i < courseCards.numCardsCreated; i++) {
+      if (courseCards[i]?.loading) return;
+    }
```

Comment:
Do you think this would be better with a filter? I guess probably not, since a filter would have you go through all course cards, whereas this stops as soon as you get one that's loading, but thought I would mention it

Reply:
Oh yeah great point, I forgot the weirdness of our courseCard type setup